src/components/DiseaseTable.tsx

```diff
@@ -103,6 +105,29 @@ const DiseaseTable = ({
         ),
     };
 
+    const handleCellInHoveredRow = (
+        record: UnpackedDiseaseCellLine,
+        index: number | undefined
+    ) => {
+        const hoveredAndDataComplete =
+            index === hoveredRowIndex && record.status === "data complete";
+        return {
+            className: hoveredAndDataComplete ? hoveredRow : "",
```

Comment:
```suggestion
            className: index === hoveredRowIndex ? hoveredRow: "", 
```
and then use the rowClassName in the `Table`
```                
rowClassName={(record) =>
                    record.status === CellLineStatus.DataComplete
                        ? dataComplete
                        : ""
                }
```

The reason for this is then you can use a css selector without any antd classes. You can have `.data-complete .hovered-row` to do your styling. Also, the `data-complete` is true regardless of hovering, so it makes sense that class would be consistent, while the hover state should be dynamic.